test/helper.rb

```diff
@@ -109,6 +109,8 @@ def md.directory?
       '/proc/net/bonding/bond0' => mockfile.call('bond0'),
       '/etc/ssh/ssh_config' => mockfile.call('ssh_config'),
       '/etc/ssh/sshd_config' => mockfile.call('sshd_config'),
+      '/etc/ssh/sshd_config_does_not_exist' => mockfile.call('sshd_config_does_not_exist'),
```

Comment:
I wonder if we should create a `fileempty` object, much like the `empty` object you see in the command mocks, so we don't have to create empty mock files.  Like this, but for files: https://github.com/chef/inspec/blob/master/test/helper.rb#L339

Reply:
Created the emptyfile object for the empty file mock. For this one I want it to point to file it cannot find.